checkbox-support/checkbox_support/helpers/timeout.py

```diff
@@ -23,13 +23,22 @@
 functions
 """
 import os
+import pickle
 import traceback
 import subprocess
 
+from queue import Empty
 from functools import partial
-from contextlib import wraps
 from unittest.mock import patch
-from multiprocessing import Process, SimpleQueue
+from contextlib import wraps, suppress
+from multiprocessing import Process, Queue
+
+
+def is_picklable(value):
```

Comment:
I had to [dive in the multiprocessing module documentation](https://docs.python.org/3/library/multiprocessing.html#pipes-and-queues) to understand why you were checking the "picklability" of things.

It might be useful to provide a one-sentence docstring to explain the need for this function (and maybe a link to the stdlib doc).

Reply:
Done